packages/kbn-es/src/cli.ts

```diff
@@ -46,7 +47,14 @@ export async function run(defaults = {}) {
       default: defaults,
     });
     const args = options._;
-    const commandName = args[0];
+    let commandName = args[0];
+
+    // Converting --serverless flag to command
```

Comment:
I'd suggest to add a bit of details to this comment, e.g. something like this:
```suggestion
    // Converting --serverless flag to command
    // `es --serverless=<projectType>` is just a shortcut for
    // `es serverless --project-type=<projectType>`
```